app/src/main/resources/liquibase-sender.xml

```diff
@@ -75,4 +75,38 @@
         </createTable>
     </changeSet>
 
+ <changeSet author="jpboane" id="20210511-1200">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <tableExists tableName="debezium_event_queue"/>
+            </not>
+        </preConditions>
+        <comment>Adding debezium_event_queue table</comment>
+
+        <createTable tableName="debezium_event_queue">
+            <column autoIncrement="true" name="id" type="BIGINT">
+                <constraints nullable="false" primaryKey="true"/>
+            </column>
+            <column name="table_name" type="VARCHAR(100)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="primary_key_id" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="identifier" type="VARCHAR(255)"/>
+            <column name="operation" type="VARCHAR(1)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="snapshot" type="BOOLEAN">
+                <constraints nullable="false"/>
+            </column>
+           	<column name="event_time_stamp" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="date_created" type="DATETIME" defaultValueComputed="CURRENT_TIMESTAMP">
```

Comment:
don't set defautValueComputed attribute, set the date from camel route

Reply:
Did you see this comment?

Reply:
Addressed now